### PR TITLE
APIS-6917: Clean up build removing invalid test configurations

### PR DESCRIPTION
**Pull request by johnsgp**

### CODE REVIEW

**Comment by johnsgp** [comment]
test this please

File: build.sbt [hunk 1]
```diff
@@ -95,30 +58,15 @@ lazy val microservice = Project(appName, file("."))
...
-  ".*definition.*",
-  "prod",
-  "testOnlyDoNotUseInAppConf",
-  "app",
-  "uk.gov.hmrc.BuildInfo"
-).mkString(";")
 
 commands ++= Seq(
-  Command.command("run-all-tests") { state => "test" :: "it:test" :: "acceptance:test" :: "sandbox:test" :: state },
+  Command.command("run-all-tests") { state => "test" :: state },
```
Comment (AndySpaven) [inline build.sbt]: Do we need all the testAll/cleanAll etc

Comment (johnsgp) [reply build.sbt]: I thought of adding them, then wondered what the value would be. I could, for consistency but no other reason.

Comment (AndySpaven) [reply build.sbt]: Just consistency is good.

**Review by AndySpaven — APPROVED** [review]

**Pull request merged**